integration/spark/src/main/scala/org/apache/spark/sql/execution/command/mutation/merge/CarbonMergeDataSetCommand.scala

```diff
@@ -267,7 +267,8 @@ case class CarbonMergeDataSetCommand(
         StructField(status_on_mergeds, IntegerType)))
     val factory =
       new SparkCarbonFileFormat().prepareWrite(sparkSession, job,
-        carbonTable.getTableInfo.getFactTable.getTableProperties.asScala.toMap, schema)
+        carbonTable.getTableInfo.getFactTable.getTableProperties.asScala
```

Comment:
I don't see any use of table properties for this intermediate (dummy table) of [tupleid, status_on_mergeds]

May be we can pass an **empty map** now for table property? 
@marchpure , @ravipesala

Reply:
or we can add new table property instead of using the source table property to sort the tupleid, status_on_mergeds if it can improve query performance.

Reply:
Tested. empty map is good.